Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM utdemir/ghc-musl:v24-ghc902 AS build
+FROM quay.io/benz0li/ghc-musl:9.6.6 AS build
```

Comment:
@DavidMazarro was the previous image deprecated?

Reply:
Not deprecated per se, but it is unmaintained. See [this comment](https://github.com/utdemir/ghc-musl/pull/42#issuecomment-1983312021) where I learned about the alternative.